tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1297,5 +1293,5 @@ main() {
 }
 
 if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
-	main $@
+	main "$@"
```

Comment:
This fix is theoretically correct in case some argument to the script contains a space. I'm not sure this happens in practice. Did you see an actual error ?

Also this script gets its arguments from `kata-deploy-binaries-in-docker.sh` which has the same _issue_ :

https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh#L139

Please fix this globally in a separate PR.